tileview/src/main/java/com/qozix/tileview/TileView.java

```diff
@@ -504,6 +504,28 @@ public View addMarker( View view, double x, double y, Float anchorX, Float ancho
     );
   }
 
+  /**
+   * Add a marker to the the TileView.  The marker can be any View.
+   * No LayoutParams are required; the View will be laid out using WRAP_CONTENT for both width and height, and positioned based on the parameters.
+   *
+   * @param view    View instance to be added to the TileView.
+   * @param x       Relative x position the View instance should be positioned at.
+   * @param y       Relative y position the View instance should be positioned at.
+   * @param anchorX The x-axis position of a marker will be offset by a number equal to the width of the marker multiplied by this value.
+   * @param anchorY The y-axis position of a marker will be offset by a number equal to the height of the marker multiplied by this value.
+   * @param typeAnchorX The x-axis position type of a marker. See POS_* flags in MarkerLayout.
```

Comment:
instead of calling this `typeAnchorX` can we try something like `xAnchorBehavior`?  same for `y` equivalents...

Reply:
Reasonable, I'll rename them.